views/dashboard.ejs

```diff
@@ -4,65 +4,34 @@
 <div class="dashboard-container">
 
 	<div class="page-header">
-  		<h1>Dashboard</h1>
-  		<p id="dashb-name">Hi <%= rider[0].name %>!</p>
-		<p id="dashb-balance">Your points balance is: <%= rider[0].pointBalance %></p>
+		<h1>Dashboard</h1>
+		<p id="dashb-name">Hi <%= rider.name %>!</p>
+		<p id="dashb-balance">Your points balance is: <%= rider.pointBalance %></p>
 	</div>
 
-<!-- Start Rewards -->
-	<div class="row">
-<!-- One Day Free Park -->
-	  <div class="col-sm-6 col-md-6">
-	    <div class="thumbnail text-center">
-		<br>
-	      <i class="fa fa-bus fa-5x" aria-hidden="true"></i>
-	      <div class="caption">
-	        <h3 class="text-center">One Day Free<br>Metro Ride</h3>
-	        <p class="text-center">5 Points</p>
-	        <p class="text-center"><a href="#" class="btn btn-default" id="oneday-redeem" role="button">Redeem</a></p>
-	      </div>
-	    </div>
-	  </div>
-<!-- One Week Free Park -->
-		<div class="col-sm-6 col-md-6">
-		  <div class="thumbnail text-center">
-		  <br>
-			<i class="fa fa-bus fa-5x" aria-hidden="true"></i>
-			<div class="caption">
-			  <h3 class="text-center">One Week Free<br>Metro Ride</h3>
-			  <p class="text-center">30 Points</p>
-			  <p class="text-center"><a href="#" class="btn btn-default" id="week-redeem" role="button">Redeem</a></p>
-			</div>
-		  </div>
-		</div>
-<!-- Uber Lyft -->
-		<div class="col-sm-6 col-md-6">
-		  <div class="thumbnail text-center">
-		  <br>
-			<i class="fa fa-car fa-5x" aria-hidden="true"></i>
-			<div class="caption">
-			  <h3 class="text-center">One Free Uber<br>or Lyft Ride</h3>
-			  <p class="text-center">(Under 5 miles)<br>50 Points</p>
-			  <p class="text-center"><a href="#" class="btn btn-default" id="uber-redeem" role="button">Redeem</a></p>
-			</div>
-		  </div>
+	<form action="/dashboard" method="post">
+
+		<div class="form-group">
+		  <input type="hidden" name="pointBalance" value="<%= rider.pointBalance %>">
 		</div>
-<!-- Car / Bike Park -->
-		<div class="col-sm-6 col-md-6">
-		  <div class="thumbnail text-center">
-		  <br>
-			<i class="fa fa-product-hunt fa-5x" aria-hidden="true"></i>
-			<div class="caption">
-			  <h3 class="text-center">1 Day Free Car Parking<br>or Bike Storage</h3>
-			  <p class="text-center"><br>50 Points</p>
-			  <p class="text-center"><a href="#" class="btn btn-default" id="parking-redeem" role="button">Redeem</a></p>
-			</div>
-		  </div>
+
+		<div class="form-group">
+
+			<input type="hidden" name="pointID" value="2">
+
 		</div>
-<!-- Img -->
-	</div>
-<!-- End Rewards -->
 
-</div>
+		<div class="form-group">
```

Comment:
@tinuola Generally, you only pass one piece of data using a select. If you need multiple pieces of data 1) don't use select; use multiple inputs including hidden inputs. 2) if you use select, pass back one piece of data that can be used to find a record in the database.